### DISCUSSION TITLE
ALT-Scann8 app does not load ALT-Scann8.json file on startup

### DISCUSSION THREAD

**Discussion by ToRa-51** [jareff-g/ALT-Scann8#94] [Q&A]
Juan,

With the latest version (1.9.12) of T-Scann8-UserInterface.py the json-file is not loaded. I know that the json file is placed in the directory (for example /home/pi/Videos/Reel14) where the frames are stored. If scanning is to be resumed the next day, the file must be moved to the /home/pi/Videos directory (in my case) to be recognized at startup. That's how I did it before, but now it doesn't happen anymore. The file also won't load if I leave it in the original directory. Can you tell if I'm doing something wrong or if it's in the app?

Greetings, Ton.

**Comment by ToRa-51** [comment]
Juan,

I did some more testing and it now appears that the json file is stored in /home/pi directory. So problem solved and it is neatly overwritten every time by the project you are working on. As it was before, careful consideration had to be given to whether the json file had already been moved or still needed to be moved.🤨
Thanks for making the software better and better.👏🏻👍🏻

Greetings, Ton.

**Reply by jareff-g** [reply]
Hello Ton,

Thanks for reporting. 

Precisely that version was fixing some wrong behavior regarding where the json file was stored. It seems originally you were suffering from it (the intention was obviously not to have the user copying it it back and forth) and not I understand it looks better for you.

For information, the file is supposed to be stores in the same folder ad the ALT-Scann9 python file resides. Is this your case?

Regards,
Juan

**Reply by ToRa-51** [reply]
Juan,

Yes, /home/pi is indeed the place where I have all the Python files and the json file is also stored there. Makes resuming scanning a lot easier if the scanning process cannot be completed in one go.

Again this time; thanks for your help.
Ton.

This afternoon I will install the latest version (1.9.22) available on Github and perhaps also test it before dinner. If I have anything to say or ask about this, I will let you know.
Where on Github should I do that?

**Reply by jareff-g** [reply]
Hi Ton,

Latest version is no longer the 1.9.22, but 1.9.23 I have just uploaded.
For any comments or questions you can open a discussion like this one (better open a new one for new topics), or an issue in case you think something is wrong.

Rgds/Juan
